src/Components/Selected/Selected.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+
+class Selected extends Component {
+  
+  setChecked = checked => {
+    for (let value of this.props.listOfTasks) {
```

Comment:
Request!
Please, extract props from this.props, for ex.:
`const { listOfTasks } = this.props;`